src/test/rpc/AMMInfo_test.cpp

```diff
@@ -75,6 +75,52 @@ class AMMInfo_test : public jtx::AMMTestBase
             }
         });
 
+        // Invalid parameters *and* invalid LP account, default API version
+        testAMM([&](AMM& ammAlice, Env&) {
+            Account bogie("bogie");
```

Comment:
`bogie` should be const. 
Consider declaring it once outside of `testAMM` and then using in each test.